server.go

```diff
@@ -1875,28 +1907,33 @@ func (sc *serverConn) runHandler(rw *responseWriter, reqCtx *app.RequestContext,
 				const size = 64 << 10
 				buf := make([]byte, size)
 				buf = buf[:runtime.Stack(buf, false)]
-				hlog.SystemLogger().Errorf("HTTP2: panic serving %v: %v\n%s", sc.conn.RemoteAddr(), e, buf)
+				err = fmt.Errorf("HTTP2: panic serving %v: %v\n%s", sc.conn.RemoteAddr(), e, buf)
+				hlog.SystemLogger().Error(err.Error())
```

Comment:
It seems that trace doesn't record the error?

Reply:
Captured by closure.